app/src/main/res/layout/feature_bubbles.xml

```diff
@@ -222,19 +222,35 @@
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 
+    <Button
+        android:id="@+id/dateButton"
+        style="@style/HedvigButton.Outlined.Negative"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/base_margin_double"
+        android:layout_marginBottom="@dimen/base_margin"
+        android:textColor="@color/off_white"
```

Comment:
We should probably not have both a style with `.Negative` and a literal `textColor` 🙂